content/docs/installation/contents.lr

```diff
@@ -44,24 +44,30 @@ Once you have those installed and have made sure that they are on your `PATH`, y
 get Lektor installed with our installation script:
 
 ```
-# curl -sf https://www.getlektor.com/install.sh | sh
+# curl -sf https://www.getlektor.com/installer.py | python3
 ```
-If you are not logged in as superuser, instead you should try this:
+
+This will attempt to install lektor in your user's `HOME`. If you want a system-wide installation, try this instead:
+
 ```
-$ curl -sf https://www.getlektor.com/install.sh | sudo sh
+$ curl -sf https://www.getlektor.com/installer.py | sudo python3
 ```
+
 If you would like to install Lektor without being prompted, set LEKTOR_SILENT before running the prior command.
 
-For Windows you can use the `command prompt`:
+For Windows you can install it directly in `Powershell`:
 
 ```
-C:\> @powershell -NoProfile -ExecutionPolicy Bypass -Command "iex ((new-object net.webclient).DownloadString('https://getlektor.com/install.ps1'))" && SET PATH=%PATH%;%LocalAppData%\lektor-cli
+PS C:\> (new-object net.webclient).DownloadString('https://www.getlektor.com/installer.py') | python
 ```
-but you can also do it directly in `Powershell`:
+
+or you can use the `command prompt` instead:
+
 ```
-PS C:\> iex ((new-object net.webclient).DownloadString('https://getlektor.com/install.ps1'))
+C:\> @powershell -NoProfile -Command "(new-object net.webclient).DownloadString('https://www.getlektor.com/installer.py') | python"
```

Comment:
I guess this assumes the user has put python.exe in their path. Does the Python installer do that by default?

Reply:
the installer has a checkbox, i don't remember if it's checked by default. but that assumption will have to hold for whatever installer (the old one too). I'll add a small blurb.